workflow/tests/hpxml_translator_test.rb

```diff
@@ -527,6 +530,7 @@ def _verify_outputs(rundir, hpxml_path, results, hpxml)
       next if err_line.include? 'DetailedSkyDiffuseModeling is chosen but not needed as either the shading transmittance for shading devices does not change throughout the year'
       next if err_line.include? 'View factors not complete'
       next if err_line.include?('CheckSimpleWAHPRatedCurvesOutputs') && err_line.include?('WaterToAirHeatPump:EquationFit') # FIXME: Check these
+      next if err_line.include? 'Target water temperature' # FIXME: Check this
```

Comment:
Still need to check this?

Reply:
I'm checking with @jmaguire1 whether we can safely ignore the "target water temperature should be less than or equal to the hot water temperature" warning.

Reply:
Sounds like these warnings are safe to ignore.